templates/artist/artist_application.html

```diff
@@ -2,6 +2,10 @@
 
 {% block content %}
     <h1>Artist Application</h1>
+    <div class="break">
+        <hr>
+    </div>
+    <div class="apply-page">
     <p>Want to be on PerDiem? Apply here.</p>
```

Comment:
nit: Please indent lines 9-17 so that it's obvious all of this is inside of the `apply-page` div.